README.md

```diff
@@ -6,14 +6,44 @@ This is a fork of the excellent [`fuser`](https://github.com/cberner/fuser) Rust
 
 This fork should be maintained in the `fuser/fork` branch of the [awslabs/mountpoint-s3 repository on GitHub](https://github.com/awslabs/mountpoint-s3/tree/main/vendor/fuser).
 
-To pull in new changes from upstream, you should fetch and rebase the changes locally and then force push to the `fuser/fork` branch (- not ideal!).
-Once the `fuser/fork` branch is as desired, create a new branch from Mountpoint's `main` branch.
-Run the [`vendor-fuser.sh` script](https://github.com/awslabs/mountpoint-s3/blob/main/vendor-fuser.sh).
-You should open a pull request with the new commit created by the script.
-This pull request is where the changes are reviewed.
-
-If you wish to add new divergent changes to this fork of Fuser,
-open a pull request on the Mountpoint repository branching from the `fuser/fork` branch and use `fuser/fork` as the base branch when creating the pull request.
+#### Sync with the origin
+
+To pull in new changes from upstream, you should fetch changes from the original upstream and rebase them locally.
```

Comment:
Maybe we should expand on why this is complex, and the high level steps we're taking?

```suggestion
To pull in new changes from upstream, we'll need to fetch the changes and rebase our patches on top, as well as maintain new versioning information.

Since the published version of this crate `mountpoint-s3-fuser` uses Git submodules and we use rebased branches to keep track of patches, we'll need to tag each version we take a dependency on in `mountpoint-s3-fuser` to avoid Git's garbage collector leading to Mountpoint commits with no available version of this code.
```